readme.md

```diff
@@ -59,8 +59,12 @@ Object to watch for changes.
 
 Type: `Function`
 
-Function that gets called anytime the object changes.
+Function that gets called anytime the object changes. Provides three args: 
+1. A path to the value that was changed
```

Comment:
Would be good to show what this looks like by logging it in the above example.

Reply:
Sounds good.